python/mxnet/gluon/contrib/estimator/estimator.py

```diff
@@ -281,7 +281,7 @@ def evaluate(self,
 
         Parameters
         ----------
-        val_data : DataLoader
+        val_data : iterable Object
             Validation data loader with data and labels.
```

Comment:
It's unclear that `data loader` refers to the `gluon.DataLoader`. It would be good to clarify that any iterable object yielding a sequence of batches is acceptable and that the `gluon.DataLoader` is one instance of such iterables.
Same applies to the fit_batch below.